apps/desktop/src/lib/branch/BranchHeader.svelte

```diff
@@ -241,8 +241,10 @@
 						{#if !$pr}
```

Comment:
Should we disable the button or just not render it at all maybe?

Reply:
Yeah let's disable it, it's how it was intended, but apparently not what came about.

Reply:
You guys forgot to approve the PR ;)